my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
Need to do
`from matplotlib import *`
or else the ``pyplot`` calls will not work